app/services/claims/value_bands.rb

```diff
@@ -3,10 +3,10 @@ class ValueBands
     Struct.new('ValueBandDefinition', :id, :name, :min, :max)
 
     VALUE_BANDS = {
-      10 => Struct::ValueBandDefinition.new(10, 'less than £25,000', 0.0, 25_000.0),
-      20 => Struct::ValueBandDefinition.new(20, '£25,001 - £100,000', 25_000.01, 100_000.0),
-      30 => Struct::ValueBandDefinition.new(30, '£100,000 - £150,000', 100_000.01, 150_000.0),
-      40 => Struct::ValueBandDefinition.new(40, 'more than £150,000', 150_000.01, 99_999_999.99)
+      10 => Struct::ValueBandDefinition.new(10, 'less than £30,000', 0.0, 30_000.0),
+      20 => Struct::ValueBandDefinition.new(20, '£30,001 - £115,000', 30_000.01, 115_000.0),
+      30 => Struct::ValueBandDefinition.new(30, '£115,001 - £175,000', 115_000.01, 175_000.0),
+      40 => Struct::ValueBandDefinition.new(40, 'more than £175,000', 175_000.01, 99_999_999.99)
```

Comment:
Nitpick: Looking at this again, I think it would make sense to change the descriptions to match the actual thresholds. It just looks a bit wrong at the moment. 

```suggestion
      10 => Struct::ValueBandDefinition.new(10, '£30,000 or less', 0.0, 30_000.0),
      20 => Struct::ValueBandDefinition.new(20, '£30,000.01 - £115,000', 30_000.01, 115_000.0),
      30 => Struct::ValueBandDefinition.new(30, '£115,000.01 - £175,000', 115_000.01, 175_000.0),
      40 => Struct::ValueBandDefinition.new(40, 'more than £175,000', 175_000.01, 99_999_999.99)
```

Although as this doesn't seem to be used and we might remove it, maybe it's irrelevant 🤷